src/storage/src/hummock/sstable/mod.rs

```diff
@@ -79,12 +79,12 @@ impl Sstable {
     pub fn new_with_data(
         id: HummockSstableId,
         meta: SstableMeta,
-        data: Bytes,
+        data: &[u8],
```

Comment:
We may be able to take `Vec<Vec<u8>>` with streaming read and save one copy of the block data. cc @ALeitert